### DISCUSSION TITLE
issue with converting x64 ISO to vmdk

### DISCUSSION THREAD

**Discussion by Smokeums** [SmoothWAN/SmoothWAN#85] [General & Help]
issue getting file to mount in esxi VM, converted x64 iso with qemu to vmdk and it seems to not want to mount, any help would be appreciated!!

![IMG_0021](https://user-images.githubusercontent.com/89883462/210111076-fda67dc6-b912-492f-b402-9b6815fd0232.JPEG)

**Comment by TalalMash** [comment]
![image](https://user-images.githubusercontent.com/96490382/210111855-2ed08273-5612-4c40-9f23-0ba095c44a49.png)
Depending on the browser scale, you can expand the downloads section for the VMDK image.

**Reply by Smokeums** [reply]
wowww can't believe I didn't see that, got it spun up and its working like a charm! what version of opnwrt is 0.99.8 based on?
thanks!

**Reply by TalalMash** [reply]
Great! It's OpenWRT 21.02 (firewall3)

**Reply by Smokeums** [reply]
great! I saw the pre-release on 22.03 and noticed better support for the Intel I1225, I have an I225 NIC that I was planning on using, but when I connect the modem on the UI it's not showing the connection, any tips on getting the I225 card to work?

**Comment by TalalMash** [comment]
Hmm odd what modem model?

**Reply by Smokeums** [reply]
Xfinity XB8, I am able to get an assigned ip from  the modem and ping an outside address but the interfaces page still shows it as a "red boxed autowan 1"

also, I am assuming I can not put all of my  ISP lines into one virtual switch in ESXi and use that as a single WAN source in Smoothwan, Smoothwan needs to see each WAN source as individual device? 

should I forgo all Virtual switches in esxi and just pass through the NIC's to the vm? or is vswitch fine?

**Reply by TalalMash** [reply]
Yes, you can use [VLAN](https://smoothwan.com/tips/) to make things easier.
